setup.py

```diff
@@ -8,6 +8,7 @@
 INSTALL_REQUIRES = [
     "Django>=2.2",
     "bleach==3.1.0",
+    "bleach-whitelist>=0.0.11",
```

Comment:
Is this a typo? I can only see 0.0.10 on pypi https://pypi.org/project/bleach-whitelist/